From c859074a98726111a95fd13447a7c24dfbde6ebb Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:27 -0400
Subject: [ethernet] i40e/i40evf: Bump pf&vf build versions

Message-id: <1409658042-22689-27-git-send-email-sassmann@redhat.com>
Patchwork-id: 90221
O-Subject: [RHEL7.1 PATCH 026/281] i40e/i40evf: Bump pf&vf build versions
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Bump i40e to 0.3.34 and i40evf to 0.9.14.

Change-ID: I6b3fb8ccf55b128d2baa4bdc20d3911ec81d4a5b
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 2062862a46b6ba35368449ec8d719cfdf467a912)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 0fc6f9d..43d391b 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -38,7 +38,7 @@ static const char i40e_driver_string[] =
 
 #define DRV_VERSION_MAJOR 0
 #define DRV_VERSION_MINOR 3
-#define DRV_VERSION_BUILD 32
+#define DRV_VERSION_BUILD 34
 #define DRV_VERSION __stringify(DRV_VERSION_MAJOR) "." \
       __stringify(DRV_VERSION_MINOR) "." \
       __stringify(DRV_VERSION_BUILD)    DRV_KERN
-- 
1.7.1